### ISSUE TITLE
Getting value on a new line instead of on the same line

### ISSUE THREAD

**Issue by designatednerd** [redundent/kotlin-xml-builder#16]
Hi! Using version 1.4.5  with `kscript` to codegen some Android resources file, and I'm wondering if there's something I'm missing to keep the values of something on the same line.

I have [a script which outputs an Android `colors.xml` file](https://github.com/bakkenbaeck/PorchPirateProtector/blob/master/code/codegen/ColorGenAndroid.kts). The value it outputs in the XML is on different lines [as seen here](https://github.com/bakkenbaeck/PorchPirateProtector/blob/master/code/android/src/main/res/values/colors.xml). Is there a way to have pretty-printing but enforce that values be on the same line so instead of 

```xml
<color name="colorPrimary">
	#05C1FD
</color>
```

I can get: 

```xml
<color name="colorPrimary">#05C1FD</color>
```

Thank you for your help!

**Comment by redundent** [comment]
Hoi @designatednerd 

At the moment, every child element is put on a new line, that includes text elements.
I'll see about enhancing the pretty print piece to allow for more control over the output. I'll add an option to keep text elements like this on a single line.

**Comment by redundent** [comment]
This has been published in version 1.5.0.

Thanks for reporting it!
